templates/vertical-full-page-map/page-config.json

```diff
@@ -64,7 +64,7 @@
     "<REPLACE ME>": { // The vertical key from your search configuration
       // "label": "", // The name of the vertical in the section header and the navigation bar
       // "verticalLimit": 15, // The result count limit for vertical search
-      // "universalLimit": 5, // The result count limit for universal search
+      // "universalLimit": 4, // The result count limit for universal search
```

Comment:
Just confirming, Rose doesn't want us to remove the comment here so that this takes effect by default, correct?

Reply:
Checked with Rose and it actually should be uncommented now, so I'll update that